pytket/extensions/aqt/backends/aqt_multi_zone.py

```diff
@@ -0,0 +1,403 @@
+        return self._backend_info
+
+    @classmethod
+    def available_devices(cls, **kwargs: Any) -> List[BackendInfo]:
+        """
+        See :py:meth:`pytket.backends.Backend.available_devices`.
+        Supported kwargs: none.
+        """
+        return []
+
+    @property
+    def required_predicates(self) -> List[Predicate]:
+        preds = [
+            NoClassicalControlPredicate(),
+            NoFastFeedforwardPredicate(),
+            NoMidMeasurePredicate(),
+            NoSymbolsPredicate(),
+            GateSetPredicate(_GATE_SET),
+        ]
+        if self._backend_info is not None:
+            preds.append(MaxNQubitsPredicate(self._backend_info.n_nodes))
+        return preds
+
+    def default_compilation_pass(self, optimisation_level: int = 1) -> BasePass:
+        assert optimisation_level in range(3)
+        if optimisation_level == 0:
+            return SequencePass(
+                [
+                    FlattenRegisters(),
+                    RenameQubitsPass(self._qm),
+                    DecomposeBoxes(),
+                    self.rebase_pass(),
+                ]
+            )
+        elif optimisation_level == 1:
+            return SequencePass(
+                [
+                    DecomposeBoxes(),
+                    SynthesiseTket(),
+                    FlattenRegisters(),
+                    RenameQubitsPass(self._qm),
+                    self.rebase_pass(),
+                    SimplifyInitial(
+                        allow_classical=False, create_all_qubits=True, xcirc=_xcirc
+                    ),
+                    EulerAngleReduction(OpType.Ry, OpType.Rx),
+                ]
+            )
+        else:
+            return SequencePass(
+                [
+                    DecomposeBoxes(),
+                    FullPeepholeOptimise(),
+                    FlattenRegisters(),
+                    RenameQubitsPass(self._qm),
+                    self.rebase_pass(),
+                    SimplifyInitial(
+                        allow_classical=False, create_all_qubits=True, xcirc=_xcirc
+                    ),
+                    EulerAngleReduction(OpType.Ry, OpType.Rx),
+                ]
+            )
+
+    @property
+    def _result_id_type(self) -> _ResultIdTuple:
+        return (str, str, str)
+
+    def process_circuits(
+        self,
+        circuits: Sequence[Circuit],
+        n_shots: Union[None, int, Sequence[Optional[int]]] = None,
+        valid_check: bool = True,
+        **kwargs: KwargTypes,
+    ) -> List[ResultHandle]:
+        """
+        See :py:meth:`pytket.backends.Backend.process_circuits`.
+        Supported kwargs: none.
+        """
+        raise NotImplementedError
+
+    def _update_cache_result(
+        self, handle: ResultHandle, result_dict: Dict[str, BackendResult]
+    ) -> None:
+        raise NotImplementedError
+
+    def circuit_status(self, handle: ResultHandle) -> CircuitStatus:
+        raise NotImplementedError
```

Comment:
Does this needs a todo?

Reply:
Have you seen this comment?

Reply:
I  would not write an explicit TODO. There is no API server now it's not clear when this will be implemented. But it's not also something I'm going to forget.